.gitlab-ci.yml

```diff
@@ -16,6 +16,7 @@ stages:
 
 build:
   stage: build
+  image: docker:18-dind
```

Comment:
When missing, it is using the pipeline scope defined (in this case `docker:git`). Good to have it explicit in the job.

However, you don't need a docker:dind image when launching another docker:dind as a service, `docker:git` should be enough and it is the one we are using in every other Docker build job in other repos.

Reply:
Right, I will fix this.